pkg/util/rand/rand.go

```diff
@@ -74,12 +74,68 @@ func Perm(n int) []int {
 // of "bad words" being formed.
 var alphanums = []rune("bcdfghjklmnpqrstvwxz0123456789")
 
+// Returns a random rune from the given rune list
+func randElem(set []rune) rune {
+	return set[Intn(len(set))]
+}
+
 // String generates a random alphanumeric string, without vowels, which is n
 // characters long.  This will panic if n is less than zero.
 func String(length int) string {
 	b := make([]rune, length)
 	for i := range b {
-		b[i] = alphanums[Intn(len(alphanums))]
+		b[i] = randElem(alphanums)
 	}
 	return string(b)
 }
+
+// Required for creating phonetic strings.
+// Letter 'l' is omitted to avoid ambiguity.
+var vowels = []rune("aeiou")
+var consonants = []rune("bcdfghjkmnpqrstvwxyz")
+var letters = []rune("abcdefghijkmnpqrstuvwxyz")
+var numbers = []rune("0123456789")
+
+// Returns a rune list of length 'length' with alternating
+// vowels and consonants. Randomly starting from any of the two.
+// If length is 1, returns any random alphabet
+func getAltRune(length int) []rune {
```

Comment:
http://thedailywtf.com/articles/The-Automated-Curse-Generator

Pretty sure this is a profanity generator. :)